python/cudaq/__init__.py

```diff
@@ -102,9 +102,9 @@
 AsyncStateResult = cudaq_runtime.AsyncStateResult
 vqe = cudaq_runtime.vqe
 draw = cudaq_runtime.draw
+translate = cudaq_runtime.translate
 
 ComplexMatrix = cudaq_runtime.ComplexMatrix
-to_qir = cudaq_runtime.get_qir
```

Comment:
This looks like a breaking change because any user code that previously used this would no longer work. Would it be possible to leave the `to_qir` method in there and just warn the user that it is deprecated for 1-2 releases? If it is too difficult to maintain both, then it is probably worth discussing at a team meeting in my opinion.

Reply:
Sure, thanks!

Reply:
restored the function and added a depreciation warning